### PR TITLE
163 query samples info

### PR DESCRIPTION
**Pull request by seaparks**
ping @potocnyj - mind looking at this and telling me if it's both a) accurate and b) at least a little more informative? trying to address https://github.com/VividCortex/docs/issues/163 without rewriting the whole pages.

### CODE REVIEW

File: content/how-to-use-vividcortex/queries.md [hunk 1]
```diff
@@ -76,8 +76,12 @@ the performance graph sums all actions into the sparkline.
 
 ## Viewing Query Samples
 
-VividCortex takes individual samples of queries probabilistically. The main chart shows the
-number of samples within the selected time range. To obtain more information about a specific
+For each query that executes, Vividcortex records the query in digested form, which removes literals
+and numeric portions of database and table names and replaces them with question-marks.
+In addition, VividCortex takes individual samples of raw queries probabilistically, leaving the literals in place
+and providing additional useful information about those queries, such as EXPLAIN plans,
+database, user, and connection information, and latency.
```
Comment (potocnyj) [inline content/how-to-use-vividcortex/queries.md]: There's an extra "and" in this sentence.

File: content/how-to-use-vividcortex/queries.md [hunk 2]
```diff
@@ -76,8 +76,12 @@ the performance graph sums all actions into the sparkline.
...
+The main chart shows the number of these samples within the selected time range. To obtain more information about a specific
```
Comment (potocnyj) [inline content/how-to-use-vividcortex/queries.md]: We actually limit the number of samples returned for a selected range (if we grab a huge time-range, it picks a random-sampling of them). I'm not sure how much detail we want to go into on that here, but it's incorrect to say that we show all samples all the time.

File: content/general-reference/samples.md [hunk 3]
```diff
@@ -34,3 +34,18 @@ If query samples are a concern for you, you should know that we can:
...
+We also will not capture the EXPLAIN plan for queries, nor metrics such as database name or user, for display in the
+Queries tab.
+
+Alternatively, we can continue to collect this data (raw queries, EXPLAIN plans, etc.) for queries
+which you whitelist based on query text or regular expression. We will discard individiual information
+about queries unless it matches the whitelist regular expression. Similarly, we can collect this data for
+all queries except those that match a blacklist regular expression.
+
+Lastly, we can capture query samples, collecting all of the information except the query text itself. This allows
+you to see information about the user, database, latency, etc. for individual samples, but we will not capture
```
Comment (potocnyj) [inline content/general-reference/samples.md]: Add a note mentioning that we include the EXPLAIN plan too here?

**Comment by seaparks** [comment]
@potocnyj I made more changes if you want to take a quick peek

**Comment by potocnyj** [comment]
:shipit: 👍

**Pull request merged**